README.md
# broadcom-manager-antix

debian/changelog
broadcom-manager-antix (16.16) unstable; urgency=medium

  * Full Portuguese

 -- anticapitalista <antix@operamail.com>  Thu, 15 Mar 2018 18:39:05 +0200

broadcom-manager-antix (16.15) unstable; urgency=medium

  * updated translations

 -- anticapitalista <antix@operamail.com>  Fri, 02 Mar 2018 21:02:34 +0200

broadcom-manager-antix (16.14) unstable; urgency=medium

  * use antix-viewer

 -- anticapitalista <antix@operamail.com>  Wed, 21 Feb 2018 16:43:00 +0200

broadcom-manager-antix (16.13) unstable; urgency=medium

  * Show in Applications menu

 -- anticapitalista <antix@operamail.com>  Sat, 07 Oct 2017 23:06:21 +0300

broadcom-manager-antix (16.12) unstable; urgency=medium

  * Removed MX branding

 -- anticapitalista <antix@operamail.com>  Tue, 14 Mar 2017 18:07:53 +0200

mx-broadcom-manager (16.11mx16+1) mx; urgency=medium

  * FR translations

 -- Adrian <adrian@mxlinux.org>  Wed, 30 Nov 2016 10:34:59 -0500

mx-broadcom-manager (16.11mx16) mx; urgency=medium

  * new translations

 -- Adrian <adrian@mxlinux.org>  Wed, 16 Nov 2016 11:41:01 -0500

mx-broadcom-manager (16.10mx16) mx; urgency=medium

  * changed URLs

 -- Adrian <adrian@mxlinux.org>  Mon, 10 Oct 2016 12:36:48 -0400

mx-broadcom-manager (16.6mx15) mx; urgency=medium

  * minor fix

 -- Adrian <adrian@mxlinux.org>  Mon, 06 Jun 2016 17:59:20 -0400

mx-broadcom-manager (15.12mx15) mx; urgency=medium

  * hide GUI when displaying help/license

 -- Adrian <adrian@mxlinux.org>  Tue, 22 Dec 2015 18:29:44 -0500

mx-broadcom-manager (15.11.1mx15) mx; urgency=medium

  * new translations

 -- Adrian <adrian@mxlinux.org>  Mon, 30 Nov 2015 11:43:04 -0500

mx-broadcom-manager (15.11mx15) mx; urgency=medium

  * fix for loading/unloading modules

 -- Adrian <adrian@mxlinux.org>  Thu, 12 Nov 2015 16:23:43 -0500

mx-broadcom-manager (15.10.1mx15) mx; urgency=medium

  * new translations

 -- Adrian <adrian@mxlinux.org>  Fri, 23 Oct 2015 11:57:46 -0400

mx-broadcom-manager (15.10mx15) mx; urgency=medium

  * ndiswrapper fix for Jessie

 -- Adrian <adrian@mxlinux.org>  Sun, 11 Oct 2015 10:19:04 -0400

mx-broadcom-manager (15.7.1mx15) mx; urgency=medium

  * change license URL

 -- Adrian <adrian@mxlinux.org>  Thu, 09 Jul 2015 17:52:04 -0400

mx-broadcom-manager (15.7mx15) mx; urgency=medium

  * update to qt5

 -- Adrian <adrian@mxlinux.org>  Tue, 30 Jun 2015 12:40:53 -0400

mx-broadcom-manager (15.6mx14) mx; urgency=medium

  * small fixes, changed version scheme

 -- Adrian <adrian@mxlinux.org>  Wed, 10 Jun 2015 10:57:02 -0400

mx-broadcom-manager (0.14.4) mx; urgency=medium

  * added blacklist/unblacklist for Broadcom drivers

 -- Adrian <adrian@mxlinux.org>  Mon, 15 Dec 2014 12:25:33 -0500

mx-broadcom-manager (0.14.3) mx; urgency=low

  * Initial release
  * 

 -- Adrian <adrian@mxlinux.org>  Sat, 02 Aug 2014 15:03:58 -0400



moc_mconfig.cpp
/****************************************************************************
** Meta object code from reading C++ file 'mconfig.h'
**
** Created by: The Qt Meta Object Compiler version 67 (Qt 5.7.1)
**
** WARNING! All changes made in this file will be lost!
*****************************************************************************/

#include "mconfig.h"
#include <QtCore/qbytearray.h>
#include <QtCore/qmetatype.h>
#if !defined(Q_MOC_OUTPUT_REVISION)
#error "The header file 'mconfig.h' doesn't include <QObject>."
#elif Q_MOC_OUTPUT_REVISION != 67
#error "This file was generated using the moc from 5.7.1. It"
#error "cannot be used with the include files from this version of Qt."
#error "(The moc has changed too much.)"
#endif

QT_BEGIN_MOC_NAMESPACE
struct qt_meta_stringdata_MConfig_t {
    QByteArrayData data[46];
    char stringdata0[1081];
};
#define QT_MOC_LITERAL(idx, ofs, len) \
    Q_STATIC_BYTE_ARRAY_DATA_HEADER_INITIALIZER_WITH_OFFSET(len, \
    qptrdiff(offsetof(qt_meta_stringdata_MConfig_t, stringdata0) + ofs \
        - idx * sizeof(QByteArrayData)) \
    )
static const qt_meta_stringdata_MConfig_t qt_meta_stringdata_MConfig = {
    {
QT_MOC_LITERAL(0, 0, 7), // "MConfig"
QT_MOC_LITERAL(1, 8, 4), // "show"
QT_MOC_LITERAL(2, 13, 0), // ""
QT_MOC_LITERAL(3, 14, 27), // "on_tabWidget_currentChanged"
QT_MOC_LITERAL(4, 42, 23), // "on_buttonCancel_clicked"
QT_MOC_LITERAL(5, 66, 22), // "on_buttonAbout_clicked"
QT_MOC_LITERAL(6, 89, 32), // "on_generalHelpPushButton_clicked"
QT_MOC_LITERAL(7, 122, 31), // "on_hwDiagnosePushButton_clicked"
QT_MOC_LITERAL(8, 154, 37), // "on_linuxDrvDiagnosePushButton..."
QT_MOC_LITERAL(9, 192, 39), // "on_windowsDrvDiagnosePushButt..."
QT_MOC_LITERAL(10, 232, 33), // "on_linuxDrvList_currentRowCha..."
QT_MOC_LITERAL(11, 266, 10), // "currentRow"
QT_MOC_LITERAL(12, 277, 38), // "on_linuxDrvBlacklistPushButto..."
QT_MOC_LITERAL(13, 316, 34), // "on_windowsDrvAddPushButton_cl..."
QT_MOC_LITERAL(14, 351, 37), // "on_windowsDrvRemovePushButton..."
QT_MOC_LITERAL(15, 389, 26), // "on_clearPingOutput_clicked"
QT_MOC_LITERAL(16, 416, 27), // "on_clearTraceOutput_clicked"
QT_MOC_LITERAL(17, 444, 27), // "on_tracerouteButton_clicked"
QT_MOC_LITERAL(18, 472, 21), // "on_pingButton_clicked"
QT_MOC_LITERAL(19, 494, 21), // "on_cancelPing_clicked"
QT_MOC_LITERAL(20, 516, 22), // "on_cancelTrace_clicked"
QT_MOC_LITERAL(21, 539, 15), // "writePingOutput"
QT_MOC_LITERAL(22, 555, 16), // "writeTraceOutput"
QT_MOC_LITERAL(23, 572, 18), // "writeInstallOutput"
QT_MOC_LITERAL(24, 591, 12), // "pingFinished"
QT_MOC_LITERAL(25, 604, 18), // "tracerouteFinished"
QT_MOC_LITERAL(26, 623, 17), // "aptUpdateFinished"
QT_MOC_LITERAL(27, 641, 15), // "installFinished"
QT_MOC_LITERAL(28, 657, 21), // "uninstallNdisFinished"
QT_MOC_LITERAL(29, 679, 35), // "on_windowsDrvList_currentRowC..."
QT_MOC_LITERAL(30, 715, 3), // "row"
QT_MOC_LITERAL(31, 719, 17), // "hwListToClipboard"
QT_MOC_LITERAL(32, 737, 21), // "hwListFullToClipboard"
QT_MOC_LITERAL(33, 759, 23), // "linuxDrvListToClipboard"
QT_MOC_LITERAL(34, 783, 27), // "linuxDrvListFullToClipboard"
QT_MOC_LITERAL(35, 811, 25), // "windowsDrvListToClipboard"
QT_MOC_LITERAL(36, 837, 29), // "windowsDrvListFullToClipboard"
QT_MOC_LITERAL(37, 867, 20), // "showContextMenuForHw"
QT_MOC_LITERAL(38, 888, 3), // "pos"
QT_MOC_LITERAL(39, 892, 26), // "showContextMenuForLinuxDrv"
QT_MOC_LITERAL(40, 919, 28), // "showContextMenuForWindowsDrv"
QT_MOC_LITERAL(41, 948, 29), // "on_installNdiswrapper_clicked"
QT_MOC_LITERAL(42, 978, 31), // "on_uninstallNdiswrapper_clicked"
QT_MOC_LITERAL(43, 1010, 20), // "on_hwUnblock_clicked"
QT_MOC_LITERAL(44, 1031, 23), // "on_linuxDrvLoad_clicked"
QT_MOC_LITERAL(45, 1055, 25) // "on_linuxDrvUnload_clicked"

    },
    "MConfig\0show\0\0on_tabWidget_currentChanged\0"
    "on_buttonCancel_clicked\0on_buttonAbout_clicked\0"
    "on_generalHelpPushButton_clicked\0"
    "on_hwDiagnosePushButton_clicked\0"
    "on_linuxDrvDiagnosePushButton_clicked\0"
    "on_windowsDrvDiagnosePushButton_clicked\0"
    "on_linuxDrvList_currentRowChanged\0"
    "currentRow\0on_linuxDrvBlacklistPushButton_clicked\0"
    "on_windowsDrvAddPushButton_clicked\0"
    "on_windowsDrvRemovePushButton_clicked\0"
    "on_clearPingOutput_clicked\0"
    "on_clearTraceOutput_clicked\0"
    "on_tracerouteButton_clicked\0"
    "on_pingButton_clicked\0on_cancelPing_clicked\0"
    "on_cancelTrace_clicked\0writePingOutput\0"
    "writeTraceOutput\0writeInstallOutput\0"
    "pingFinished\0tracerouteFinished\0"
    "aptUpdateFinished\0installFinished\0"
    "uninstallNdisFinished\0"
    "on_windowsDrvList_currentRowChanged\0"
    "row\0hwListToClipboard\0hwListFullToClipboard\0"
    "linuxDrvListToClipboard\0"
    "linuxDrvListFullToClipboard\0"
    "windowsDrvListToClipboard\0"
    "windowsDrvListFullToClipboard\0"
    "showContextMenuForHw\0pos\0"
    "showContextMenuForLinuxDrv\0"
    "showContextMenuForWindowsDrv\0"
    "on_installNdiswrapper_clicked\0"
    "on_uninstallNdiswrapper_clicked\0"
    "on_hwUnblock_clicked\0on_linuxDrvLoad_clicked\0"
    "on_linuxDrvUnload_clicked"
};
#undef QT_MOC_LITERAL

static const uint qt_meta_data_MConfig[] = {

 // content:
       7,       // revision
       0,       // classname
       0,    0, // classinfo
      41,   14, // methods
       0,    0, // properties
       0,    0, // enums/sets
       0,    0, // constructors
       0,       // flags
       0,       // signalCount

 // slots: name, argc, parameters, tag, flags
       1,    0,  219,    2, 0x0a /* Public */,
       3,    0,  220,    2, 0x0a /* Public */,
       4,    0,  221,    2, 0x0a /* Public */,
       5,    0,  222,    2, 0x0a /* Public */,
       6,    0,  223,    2, 0x0a /* Public */,
       7,    0,  224,    2, 0x0a /* Public */,
       8,    0,  225,    2, 0x0a /* Public */,
       9,    0,  226,    2, 0x0a /* Public */,
      10,    1,  227,    2, 0x0a /* Public */,
      12,    0,  230,    2, 0x0a /* Public */,
      13,    0,  231,    2, 0x0a /* Public */,
      14,    0,  232,    2, 0x0a /* Public */,
      15,    0,  233,    2, 0x0a /* Public */,
      16,    0,  234,    2, 0x0a /* Public */,
      17,    0,  235,    2, 0x0a /* Public */,
      18,    0,  236,    2, 0x0a /* Public */,
      19,    0,  237,    2, 0x0a /* Public */,
      20,    0,  238,    2, 0x0a /* Public */,
      21,    0,  239,    2, 0x0a /* Public */,
      22,    0,  240,    2, 0x0a /* Public */,
      23,    0,  241,    2, 0x0a /* Public */,
      24,    0,  242,    2, 0x0a /* Public */,
      25,    0,  243,    2, 0x0a /* Public */,
      26,    0,  244,    2, 0x0a /* Public */,
      27,    1,  245,    2, 0x0a /* Public */,
      28,    1,  248,    2, 0x0a /* Public */,
      29,    1,  251,    2, 0x0a /* Public */,
      31,    0,  254,    2, 0x0a /* Public */,
      32,    0,  255,    2, 0x0a /* Public */,
      33,    0,  256,    2, 0x0a /* Public */,
      34,    0,  257,    2, 0x0a /* Public */,
      35,    0,  258,    2, 0x0a /* Public */,
      36,    0,  259,    2, 0x0a /* Public */,
      37,    1,  260,    2, 0x0a /* Public */,
      39,    1,  263,    2, 0x0a /* Public */,
      40,    1,  266,    2, 0x0a /* Public */,
      41,    0,  269,    2, 0x08 /* Private */,
      42,    0,  270,    2, 0x08 /* Private */,
      43,    0,  271,    2, 0x08 /* Private */,
      44,    0,  272,    2, 0x08 /* Private */,
      45,    0,  273,    2, 0x08 /* Private */,

 // slots: parameters
    QMetaType::Void,
    QMetaType::Void,
    QMetaType::Void,
    QMetaType::Void,
    QMetaType::Void,
    QMetaType::Void,
    QMetaType::Void,
    QMetaType::Void,
    QMetaType::Void, QMetaType::Int,   11,
    QMetaType::Void,
    QMetaType::Void,
    QMetaType::Void,
    QMetaType::Void,
    QMetaType::Void,
    QMetaType::Void,
    QMetaType::Void,
    QMetaType::Void,
    QMetaType::Void,
    QMetaType::Void,
    QMetaType::Void,
    QMetaType::Void,
    QMetaType::Void,
    QMetaType::Void,
    QMetaType::Void,
    QMetaType::Void, QMetaType::Int,    2,
    QMetaType::Void, QMetaType::Int,    2,
    QMetaType::Void, QMetaType::Int,   30,
    QMetaType::Void,
    QMetaType::Void,
    QMetaType::Void,
    QMetaType::Void,
    QMetaType::Void,
    QMetaType::Void,
    QMetaType::Void, QMetaType::QPoint,   38,
    QMetaType::Void, QMetaType::QPoint,   38,
    QMetaType::Void, QMetaType::QPoint,   38,
    QMetaType::Void,
    QMetaType::Void,
    QMetaType::Void,
    QMetaType::Void,
    QMetaType::Void,

       0        // eod
};

void MConfig::qt_static_metacall(QObject *_o, QMetaObject::Call _c, int _id, void **_a)
{
    if (_c == QMetaObject::InvokeMetaMethod) {
        MConfig *_t = static_cast<MConfig *>(_o);
        Q_UNUSED(_t)
        switch (_id) {
        case 0: _t->show(); break;
        case 1: _t->on_tabWidget_currentChanged(); break;
        case 2: _t->on_buttonCancel_clicked(); break;
        case 3: _t->on_buttonAbout_clicked(); break;
        case 4: _t->on_generalHelpPushButton_clicked(); break;
        case 5: _t->on_hwDiagnosePushButton_clicked(); break;
        case 6: _t->on_linuxDrvDiagnosePushButton_clicked(); break;
        case 7: _t->on_windowsDrvDiagnosePushButton_clicked(); break;
        case 8: _t->on_linuxDrvList_currentRowChanged((*reinterpret_cast< int(*)>(_a[1]))); break;
        case 9: _t->on_linuxDrvBlacklistPushButton_clicked(); break;
        case 10: _t->on_windowsDrvAddPushButton_clicked(); break;
        case 11: _t->on_windowsDrvRemovePushButton_clicked(); break;
        case 12: _t->on_clearPingOutput_clicked(); break;
        case 13: _t->on_clearTraceOutput_clicked(); break;
        case 14: _t->on_tracerouteButton_clicked(); break;
        case 15: _t->on_pingButton_clicked(); break;
        case 16: _t->on_cancelPing_clicked(); break;
        case 17: _t->on_cancelTrace_clicked(); break;
        case 18: _t->writePingOutput(); break;
        case 19: _t->writeTraceOutput(); break;
        case 20: _t->writeInstallOutput(); break;
        case 21: _t->pingFinished(); break;
        case 22: _t->tracerouteFinished(); break;
        case 23: _t->aptUpdateFinished(); break;
        case 24: _t->installFinished((*reinterpret_cast< int(*)>(_a[1]))); break;
        case 25: _t->uninstallNdisFinished((*reinterpret_cast< int(*)>(_a[1]))); break;
        case 26: _t->on_windowsDrvList_currentRowChanged((*reinterpret_cast< int(*)>(_a[1]))); break;
        case 27: _t->hwListToClipboard(); break;
        case 28: _t->hwListFullToClipboard(); break;
        case 29: _t->linuxDrvListToClipboard(); break;
        case 30: _t->linuxDrvListFullToClipboard(); break;
        case 31: _t->windowsDrvListToClipboard(); break;
        case 32: _t->windowsDrvListFullToClipboard(); break;
        case 33: _t->showContextMenuForHw((*reinterpret_cast< const QPoint(*)>(_a[1]))); break;
        case 34: _t->showContextMenuForLinuxDrv((*reinterpret_cast< const QPoint(*)>(_a[1]))); break;
        case 35: _t->showContextMenuForWindowsDrv((*reinterpret_cast< const QPoint(*)>(_a[1]))); break;
        case 36: _t->on_installNdiswrapper_clicked(); break;
        case 37: _t->on_uninstallNdiswrapper_clicked(); break;
        case 38: _t->on_hwUnblock_clicked(); break;
        case 39: _t->on_linuxDrvLoad_clicked(); break;
        case 40: _t->on_linuxDrvUnload_clicked(); break;
        default: ;
        }
    }
}

const QMetaObject MConfig::staticMetaObject = {
    { &QDialog::staticMetaObject, qt_meta_stringdata_MConfig.data,
      qt_meta_data_MConfig,  qt_static_metacall, Q_NULLPTR, Q_NULLPTR}
};


const QMetaObject *MConfig::metaObject() const
{
    return QObject::d_ptr->metaObject ? QObject::d_ptr->dynamicMetaObject() : &staticMetaObject;
}

void *MConfig::qt_metacast(const char *_clname)
{
    if (!_clname) return Q_NULLPTR;
    if (!strcmp(_clname, qt_meta_stringdata_MConfig.stringdata0))
        return static_cast<void*>(const_cast< MConfig*>(this));
    if (!strcmp(_clname, "Ui::MEConfig"))
        return static_cast< Ui::MEConfig*>(const_cast< MConfig*>(this));
    return QDialog::qt_metacast(_clname);
}

int MConfig::qt_metacall(QMetaObject::Call _c, int _id, void **_a)
{
    _id = QDialog::qt_metacall(_c, _id, _a);
    if (_id < 0)
        return _id;
    if (_c == QMetaObject::InvokeMetaMethod) {
        if (_id < 41)
            qt_static_metacall(this, _c, _id, _a);
        _id -= 41;
    } else if (_c == QMetaObject::RegisterMethodArgumentMetaType) {
        if (_id < 41)
            *reinterpret_cast<int*>(_a[0]) = -1;
        _id -= 41;
    }
    return _id;
}
QT_END_MOC_NAMESPACE
